src/components/LicenseRevokeModal/index.jsx

```diff
@@ -79,16 +72,39 @@ class LicenseRevokeModal extends React.Component {
     const {
       user,
       submitFailed,
+      subscriptionPlan,
+      licenseOverview,
     } = this.props;
 
+    const { revocations } = subscriptionPlan;
+
     return (
       <React.Fragment>
-        {submitFailed && this.renderErrorMessage()}
-        <div className="license-details mb-4">
+        <StatusAlert
+          alertType="warning"
+          message={
+            <p className="m-0">
+              You have already revoked {revocations.applied} licenses. You
+              have {revocations.remaining} left on your plan.
```

Comment:
Nit: "You have N revocations left on your plan" (?)

Reply:
![image](https://user-images.githubusercontent.com/2828721/95488751-ea2a8700-0963-11eb-8964-bfc3bc2741a0.png)

The wireframe has "You have N more left on your plan." Looks like I missed the word "more" but I agree it feels like we need the word "revocation" in there as well.